tests/default.nf.test.snap

```diff
@@ -1,21 +1,21 @@
 {
     "-profile test": {
         "content": [
-            "bracken/database-bracken.tar.gz - minimum file size: true",
+            "bracken/database-bracken.tar.gz - minimum file size: false",
```

Comment:
Hrm need to check all of these, they should all be set to true (might just need to reduce specified size a bit)